source/grid/reference_cell.cc

```diff
@@ -181,6 +181,52 @@ namespace ReferenceCell
     return Quadrature<dim>(); // never reached
   }
 
+  template <int dim>
+  Quadrature<dim> &
+  get_nodal_type_quadrature(const Type &reference_cell)
+  {
+    AssertDimension(dim, get_dimension(reference_cell));
+
+    const auto create_quadrature = [](const Type &reference_cell) {
+      Triangulation<dim> tria;
+      make_triangulation(reference_cell, tria);
+
+      std::vector<Point<dim>> points(tria.n_vertices());
+
+      for (unsigned int v = 0; v < points.size(); ++v)
+        points[v] = tria.get_vertices()[v];
+
+      return Quadrature<dim>(points);
```

Comment:
return 
```suggestion
      return Quadrature<dim>(tria.get_vertices());
```

Reply:
You are right! How stupid of me!

Reply:
I do this kind of stuff all the time... :D